python/tests/test_pyraphtory.py

```diff
@@ -161,3 +164,10 @@ def test_mutable_integer(self):
         i = pyraphtory.input.MutableInteger("test", 2)
         self.assertEqual(i.value(), 2)
         self.assertEqual(i.key(), "test")
+
```

Comment:
Can we change this to a local sqlite instance which we only have to download once